esprima.js

```diff
@@ -4696,12 +4804,20 @@
                 token = lookahead;
                 isStatic = false;
                 computed = match('[');
-                key = parseObjectPropertyKey();
-                if (key.name === 'static' && lookaheadPropertyName()) {
-                    token = lookahead;
-                    isStatic = true;
-                    computed = match('[');
+                if (match('*')) {
+                    lex();
```

Comment:
This fails on `*[computedKey](){}`. The handling of `*` should probably happen before `computed = match('[');`.